.github/workflows/test.yml

```diff
@@ -0,0 +1,82 @@
+on:
+  - push
+
+jobs:
+  test-clj:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3.0.2
+
+      - name: Setup Java
+        uses: actions/setup-java@v3.4.0
+        with:
+          distribution: temurin
+          java-version: 17
```

Comment:
Have you considered using a matrix syntax? Running it on all the LTS JDK versions would seem reasonable.